src/Umbraco.Infrastructure/Models/ContentEditing/UserInvite.cs

```diff
@@ -33,7 +33,8 @@ public IEnumerable<ValidationResult> Validate(ValidationContext validationContex
             if (UserGroups.Any() == false)
                 yield return new ValidationResult("A user must be assigned to at least one group", new[] { nameof(UserGroups) });
 
-            if (Current.Configs.Security().UsernameIsEmail == false && Username.IsNullOrWhiteSpace())
+            // TODO: this will need another way of retrieving this setting if and when Configs are removed from Current.
```

Comment:
You can remove this `TODO` now @bergmania (now you've removed `Current.Configs`.

Reply:
These TODOs are still required, as I just replaced `Current.Configs` with their specific config class on `Current`

Reply:
Not sure.  Just to be clear, I added these, they weren't there before.  And I added them to note that we'd need to changes this when `Current.Configs` is no longer available.  And now it's not.  So it could be you want to add a new TODO about refactoring from use of `Current` per so (i.e. no longer using service location if we can avoid it), but that wasn't there before and wasn't what I intended with the comment..